handlers.go

```diff
@@ -195,20 +205,20 @@ func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 			}
 			sigresps[i].SignedFile = base64.StdEncoding.EncodeToString(signedfile)
 			// calculate a hash of the input to store in the signing logs
-			md := sha256.New()
-			md.Write(input)
-			hashlog = fmt.Sprintf("%X", md.Sum(nil))
+			inputHash = hashSHA256AsHex(input)
+			outputHash = hashSHA256AsHex(signedfile)
 		}
 		log.WithFields(log.Fields{
-			"rid":        rid,
-			"options":    sigreq.Options,
-			"mode":       sigresps[i].Mode,
-			"ref":        sigresps[i].Ref,
-			"type":       sigresps[i].Type,
-			"signer_id":  sigresps[i].SignerID,
-			"input_hash": hashlog,
-			"user_id":    userid,
-			"t":          int32(time.Since(starttime) / time.Millisecond), //  request processing time in ms
+			"rid":         rid,
+			"options":     sigreq.Options,
+			"mode":        sigresps[i].Mode,
+			"ref":         sigresps[i].Ref,
+			"type":        sigresps[i].Type,
+			"signer_id":   sigresps[i].SignerID,
+			"input_hash":  inputHash,
+			"output_hash": outputHash,
```

Comment:
`input_hash` is common to all three endpoints, but `output_hash` is only set by `/sign/file` and I can see us going through the logs wondering why `output_hash` is empty without noticing the call is for `/sign/hash` or `/sign/data`. I'm wondering if renaming this field to `signed_file_hash` would reduce this confusion. What do you think?

Reply:
I was thinking we'd want to log output hash for `/hash` and `/data` and use a standard field name, but was running into errors and punted on the idea thinking we could add them later if desired. What about an explicit string like `"unimplemented"` so it's clearer in the logs?

I can take another shot at `/data` since I think that'd be helpful.